README.md

```diff
@@ -0,0 +1,57 @@
+# Today Migration Plugin #
+
+Provides a series of wp cli tasks for manipulating data used by the old Today-Bootstrap theme to work with the new theme and associated plugins.
+
+
+## Description ##
+
+Provides a series of wp cli tasks for manipulating data used by the old Today-Bootstrap theme to work with the new theme and associated plugins.
```

Comment:
Do we need to have this description section if the above description is the same?

Reply:
The second description in the readme.txt file gets used by WordPress when it displays the plugin in the list menu. Normally, these two descriptions would be different, the top one being a little longer and more descriptive. But since this plugin will probably get deleted after we use it once, I didn't bother.